framework/i18n/Formatter.php

```diff
@@ -876,6 +876,10 @@ public function asDuration($value, $implodeString = ', ', $negativeSign = '-')
         if ($interval->s > 0) {
             $parts[] = Yii::t('yii', '{delta, plural, =1{1 second} other{# seconds}}', ['delta' => $interval->s], $this->locale);
         }
+        if ($interval->s === 0 && empty($parts)) {
+            $parts[] = Yii::t('yii', '{delta} seconds', ['delta' => $interval->s], $this->locale);
```

Comment:
I'd prefer using the message from previous condition (`'{delta, plural, =1{1 second} other{# seconds}}'`) in order to avoid unnecessary translations update.